codersdk/flags.go

```diff
@@ -10,57 +10,57 @@ import (
-	CacheDir                         StringFlag      `json:"cache_dir"`
-	InMemoryDatabase                 BoolFlag        `json:"in_memory_database"`
-	ProvisionerDaemonCount           IntFlag         `json:"provisioner_daemon_count"`
-	PostgresURL                      StringFlag      `json:"postgres_url"`
-	OAuth2GithubClientID             StringFlag      `json:"oauth2_github_client_id"`
-	OAuth2GithubClientSecret         StringFlag      `json:"oauth2_github_client_secret"`
-	OAuth2GithubAllowedOrganizations StringArrayFlag `json:"oauth2_github_allowed_organizations"`
-	OAuth2GithubAllowedTeams         StringArrayFlag `json:"oauth2_github_allowed_teams"`
-	OAuth2GithubAllowSignups         BoolFlag        `json:"oauth2_github_allow_signups"`
-	OAuth2GithubEnterpriseBaseURL    StringFlag      `json:"oauth2_github_enterprise_base_url"`
-	OIDCAllowSignups                 BoolFlag        `json:"oidc_allow_signups"`
-	OIDCClientID                     StringFlag      `json:"oidc_client_id"`
-	OIDCClientSecret                 StringFlag      `json:"oidc_cliet_secret"`
-	OIDCEmailDomain                  StringFlag      `json:"oidc_email_domain"`
-	OIDCIssuerURL                    StringFlag      `json:"oidc_issuer_url"`
-	OIDCScopes                       StringArrayFlag `json:"oidc_scopes"`
-	TelemetryEnable                  BoolFlag        `json:"telemetry_enable"`
-	TelemetryTraceEnable             BoolFlag        `json:"telemetry_trace_enable"`
-	TelemetryURL                     StringFlag      `json:"telemetry_url"`
-	TLSEnable                        BoolFlag        `json:"tls_enable"`
-	TLSCertFiles                     StringArrayFlag `json:"tls_cert_files"`
-	TLSClientCAFile                  StringFlag      `json:"tls_client_ca_file"`
-	TLSClientAuth                    StringFlag      `json:"tls_client_auth"`
-	TLSKeyFiles                      StringArrayFlag `json:"tls_key_tiles"`
-	TLSMinVersion                    StringFlag      `json:"tls_min_version"`
-	TraceEnable                      BoolFlag        `json:"trace_enable"`
-	SecureAuthCookie                 BoolFlag        `json:"secure_auth_cookie"`
-	SSHKeygenAlgorithm               StringFlag      `json:"ssh_keygen_algorithm"`
-	AutoImportTemplates              StringArrayFlag `json:"auto_import_templates"`
-	MetricsCacheRefreshInterval      DurationFlag    `json:"metrics_cache_refresh_interval"`
-	AgentStatRefreshInterval         DurationFlag    `json:"agent_stat_refresh_interval"`
-	Verbose                          BoolFlag        `json:"verbose"`
-	AuditLogging                     BoolFlag        `json:"audit_logging"`
-	BrowserOnly                      BoolFlag        `json:"browser_only"`
-	SCIMAuthHeader                   StringFlag      `json:"scim_auth_header"`
-	UserWorkspaceQuota               IntFlag         `json:"user_workspace_quota"`
+	AccessURL                        *StringFlag      `json:"access_url" typescript:",notnull"`
```

Comment:
Are we not planning on returning this from the backend? I thought we had a `coderd` endpoint with it for this reason... then the frontend could display these values.

Reply:
What do you mean here exactly? This type is properly generated and the endpoint does return this struct and will be consumed by the frontend shortly. Are you referencing the `typescript` tag? We just do this so that the type is generated in typescript as `field: type` instead of `field?: type`.

Reply:
Ohhhhhhh